src/js/plugins/monitor.js

```diff
@@ -51,69 +51,69 @@ class Monitor extends Plugin {
         mainDiv.innerHTML = `<div class="title grid__col grid__col--8-of-8"></div>
         <table class="title grid__col grid__col--8-of-8">
         <tr><td><div class="text grid__col grid__col--6-of-8">Observables</div></td>
-        <td><div class="text grid__col grid__col--6-of-8">
-            <select id="observables" style="width: fit-content;"></select>                   </div></td><td rowspan="4">
+        <td><div class="text grid__col grid__col--6-of-8" style="width: 400px;">
```

Comment:
This project uses a mini grid system, setting the fixed width to the element is incorrect.

Please use the `grid__col--6-of-8` values accordingly. Every row can have 8 columns and you can divide the elements per row accordingly. There are plenty of examples on other plugins on how the grid system is used.

E.g. you can have the label in a 2 width column and the value on the remaining 6 places. The CSS grid system will dynamically handle the widths for you based on the user's viewport.

Reply:
For example the webkit plugin does this:
```
            <div class="label grid__col grid__col--2-of-8">URL presets</div>
            <div class="text grid__col grid__col--6-of-8">
                <select id="{{callsign}}_linkPresets"></select>
            </div>
```

The first div creates a 2 column entry and then the value is entered in the remaining 6 out of an 8 column total row. All you need to update is the grid__col values accordingly.